test/core/models/ocelot/MetaSpec.scala

```diff
@@ -79,7 +79,7 @@ class MetaSpec extends BaseSpec {
       result shouldBe validModelWithOptionalProperties
     }
 
-    missingJsObjectAttrTests[Meta](validJsonWithOptionalProperties, List("titlePhrase", "processCode", "version"))
+    missingJsObjectAttrTests[Meta](validJsonWithOptionalProperties, List("titlePhrase", "processCode", "version", "lastAuthor", "lastUpdate", "filename"))
```

Comment:
As with the backend "processCode" is a mandatory attribute and as such should not be excluded from the property tests.